### DISCUSSION TITLE
Light theme with a slightly darkened background

### DISCUSSION THREAD

**Discussion by monochromer** [primer/github-vscode-theme#309] [Ideas]
How about adding a background like in your code snippets in markdown?
For example:
```javascript
function thunkify(fn) {
  return function() {
    var args = [].slice.call( arguments );
    return function(cb) {
      args.push( cb );
      return fn.apply( null, args );
    };
  };
}
```
The pure white color makes eyes more tired.

**Comment by ianyepan** [comment]
You can put preferences and personal colour tweaks in your settings.json

**Comment by monochromer** [comment]
> You can put preferences and personal colour tweaks in your settings.json

Perhaps it would be useful not only for me

**Comment by clo4** [comment]
> You can put preferences and personal colour tweaks in your settings.json

For reference, here's an example `settings.json` .

```jsonc
// Merge this with your current config (don't copy the root object)
{
  "workbench.colorCustomizations": {
    "[GitHub Light]": {
      // This is slightly darker than GitHub code blocks for better contrast with surrounding UI.
      // Current background color of code blocks (July, 2020):  #f6f8fa
      "editor.background": "#f3f5f7"
    }
  }
}
```

To use it: <kbd>Ctrl / Cmd</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>, then begin typing `Preferences: Open Settings (JSON)`

> Perhaps it would be useful not only for me

For sure! Personally I prefer the default, but there's definitely a set of people that prefer a slightly darker editor background. Modifying it yourself is probably your best option for now.
